core/domain/state_domain.py

```diff
@@ -1234,9 +2192,11 @@ class InteractionCustomizationArg(translation_domain.BaseTranslatableObject):
     """
 
     # Here we use type Any because values in schema dictionary can be of type
-    # str, List, Dict and other types too.
+    # str, List, Dict and other types too. We also use type Any for `value`
+    # because it can be of type SubtitledHtmlDict, SubtitledUnicodeDict
+    # and so on.
```

Comment:
What are the so on types? You can always still use `Union[SubtitledHtmlDict, SubtitledUnicodeDict, ...]`

Reply:
Updated